DEVELOPMENT.md

```diff
@@ -44,11 +47,16 @@ The Batch Analysis service relies on a Postgresql database server. There are two
 - Convenient app setup:
   Install [the convenient app](https://postgresapp.com/), and create a database called `bio.terra.batchanalysis`.
 
+Make sure to use the correct version, as noted above.
+
 #### Initialize your database:
 ```sh
 psql -h 127.0.0.1 -U postgres -f ./common/postgres-init.sql
 ```
-***N.B.*** If you used **the convenient app**, you should run `psql` as `/Applications/Postgres.app/Contents/Versions/latest/bin/psql`
+***N.B.*** If you used **the convenient app**, you should run `psql` as `/Applications/Postgres.app/Contents/Versions/13/bin/psql`. To add Postgres commands to your path, run the following command:
+```sh
+sudo mkdir -p /etc/paths.d && echo /Applications/Postgres.app/Contents/Versions/13/bin | sudo tee /etc/paths.d/postgresapp
```

Comment:
Huh, TIL about this. I've always just added `PATH=$PATH:.....` into bashrc files before

Reply:
Copied straight from [Postgres setup](https://postgresapp.com/). I usually do the same thing 🤷‍♂️